charts/descheduler/templates/cronjob.yaml

```diff
@@ -51,6 +51,10 @@ spec:
           affinity:
             {{- toYaml . | nindent 12 }}
           {{- end }}
+          {{- with .Values.topologySpreadConstraints }}
+          topologySpreadConstraints:
+            {{- toYaml . | nindent 12 }}
+          {{- end }}
```

Comment:
Not sure of the value of adding this here, thoughts?
```suggestion
```

Reply:
I actually copy/pasted and adapted `affinity` so I included this here. I agree it may not be pertinent but I added it for consistency